cfme/tests/control/test_alerts_misc.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import fauxfactory
+from utils.update import update
+from cfme.control import explorer
+import cfme.fixtures.pytest_selenium as sel
+
+
+@pytest.mark.meta(blockers=[1303645], automates=[1303645])
+def test_control_alert_copy(soft_assert):
+    alert = explorer.Alert(
+        fauxfactory.gen_alphanumeric(), timeline_event=True, driving_event="Hourly Timer"
+    )
+    alert.create()
+    soft_assert(alert.exists, "The alert {} does not exist!".format(
+        alert.description
+    ))
+    alert_copy = alert.copy()
+    soft_assert(alert_copy.exists, "The alert {} does not exist!".format(
+        alert_copy.description
+    ))
+    alert_copy.delete()
+    soft_assert(not alert_copy.exists, "The alert {} exists!".format(
+        alert_copy.description
+    ))
+    alert.delete()
+    soft_assert(not alert.exists, "The alert {} exists!".format(
+        alert.description
+    ))
+
+
+def test_alert_profile_crud(soft_assert):
+    alert = explorer.Alert(
+        fauxfactory.gen_alphanumeric(),
+        timeline_event=True,
+        driving_event="Hourly Timer"
+    )
+    alert.create()
+    alert_profile = explorer.VMInstanceAlertProfile(
+        fauxfactory.gen_alphanumeric(),
+        [alert]
+    )
+    alert_profile.create()
+    soft_assert(alert_profile.exists, "The alert profile {} does not exist!".format(
+        alert_profile.description
+    ))
+    with update(alert_profile):
+        alert_profile.notes = "Modified!"
+    sel.force_navigate("%s_alert_profile_edit" % alert_profile.PREFIX,
+                       context={"alert_profile_name": alert_profile.description})
+    soft_assert(
+        sel.text(
+            alert_profile.form.notes) == "Modified!", "Modification failed!"
+    )
+    alert_profile.delete()
+    soft_assert(not alert_profile.exists, "The alert {} exists!".format(
+        alert_profile.description
+    ))
+
+
+def test_vm_control_policy_copy(soft_assert):
+    policy = explorer.VMControlPolicy(fauxfactory.gen_alphanumeric())
+    policy.create()
```

Comment:
This should be a `fixture` or rather a `yield_fixture` that would include the `policy.delete()` that is ran on the last line of this test.